components/gravityforms/class-gf-convertkit.php

```diff
@@ -250,9 +250,12 @@ public function get_column_value_form_id( $feed ) {
 		$form_id = rgars( $feed, 'meta/form_id' );
 
 		if ( is_array( $forms ) && isset( $forms[ $form_id ] ) ) {
-			$form = $forms[ $form_id ];
+			$form_url = "https://app.convertkit.com/forms/designers/$form_id/edit";
```

Comment:
Could we use a constant for the string?

Reply:
Do you mean for the API endpoint URL?

Reply:
Yeah, sorry, I wasn't clear. Basically, if we use constant here instead of `https://app.convertkit.com/`, it will make it easy to make changes if we ever need to. It's also usually best practice to avoid inline string when possible.

Reply:
Yep gotcha, makes sense. I'll update that and refactor other relevant places as well 👌